portal-api/src/v1/controllers/utilisation-report-service/due-report-periods.controller.js

```diff
@@ -9,7 +9,9 @@ const getDueReportPeriodsByBankId = async (req, res) => {
   try {
     const { bankId } = req.params;
 
-    const notReceivedReports = await api.getUtilisationReports(bankId, { reportStatuses: [UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED] });
+    const notReceivedReports = (await api.getUtilisationReports(bankId)).filter(
+      (report) => report.status === UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED,
+    );
```

Comment:
Should we be using the `excludeNotUploaded` param here?

Reply:
Wouldn't that case be handled by this filter?
```
utilisationReportDetailsFilter.azureFileInfo = { $not: { $eq: null } };
```

Reply:
Actually, we shouldn't be using that query here. `excludeNotUploaded` will remove reports in the `REPORT_NOT_RECEIVED` state, but we only want reports in this state